superset/utils/core.py

```diff
@@ -1020,6 +1021,7 @@ def send_mime_email(
     if smtp_user and smtp_password:
         smtp.login(smtp_user, smtp_password)
     logger.debug("Sent an email to %s", str(e_to))
+    logger.info("Sent email notification content is %s", mime_msg.as_string())
```

Comment:
don't think we should log the mime_msg itself

Reply:
ok, I think this one we can actually delete. Ty